SmartCity/api/jwt.py

```diff
@@ -3,16 +3,16 @@
 import urllib3
 
 
-JWT_SECRET = None
+JWT_SECRET = "1"
 
 
 def read_payload(request):
-    return jwt.decode(request.cookies.get("accessToken"), JWT_SECRET, algorithms=["HS256"])
+    return jwt.decode(request.COOKIES.get("accessToken"), JWT_SECRET, algorithms=["HS256"])
```

Comment:
Ist das gewollt, dass die jetzt in CapsLock geschrieben sind?

Reply:
Soll ich also noch nicht mergen?

Reply:
Jetzt schon :)